src/app/visual/edit-stop-when-loses-focus/edit-stop-when-loses-focus.component.html

```diff
@@ -0,0 +1,55 @@
+
+<sky-docs-demo-control-panel-theme
+  (themeSettingsChange)="themeSettingsChange($event)"
+>
+</sky-docs-demo-control-panel-theme>
+
+<ng-container
```

Comment:
What is the purpose of this container? We are showing the grid either way so the if statement seems like a noop.